string filename
uint8 record_rate
---
uint8 isError
